README.md

```diff
@@ -113,7 +113,7 @@ Add [`kotlinx-coroutines-android`](ui/kotlinx-coroutines-android)
 module as dependency when using `kotlinx.coroutines` on Android:
 
 ```groovy
-implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-android:0.25.3'
+implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-android:0.26.0'
 ```
 
 This gives you access to Android [UI](https://kotlin.github.io/kotlinx.coroutines/kotlinx-coroutines-android/kotlinx.coroutines.experimental.android/-u-i.html)
```

Comment:
Oops... I've missed this mention of Android's `UI`... Need to be replaced with `Dispatchers.Main`